### ISSUE TITLE
Transition.In not working when animateNextTransition() called inside useEffect after first Render

### ISSUE THREAD

**Issue by tonilxm** [software-mansion/react-native-reanimated#796]
Transition.In not working when animateNextTransition() called inside useEffect after first Render.
Version :
"react-native": "0.62.2",
"react-native-reanimated": "^1.8.0"

![2](https://user-images.githubusercontent.com/4486196/82491815-633d5500-9b0f-11ea-9e83-307371c9c0fa.gif)

Here is my code:
```js
import React, {useRef, useEffect, useState} from 'react';
import {
  SafeAreaView,
  StyleSheet,
  ScrollView,
  View,
  Text,
  Button,
  StatusBar,
} from 'react-native';
import {Transitioning, Transition} from 'react-native-reanimated';

declare const global: {HermesInternal: null | {}};

const App = () => {
  const [refreshed, setRefreshed] = useState(0);
  const ref = useRef();

  useEffect(() => {
    animate();
  }, []);

  const handleRefreshed = () => {
    console.log('refreshed');
    setRefreshed(refreshed + 1);
    ref.current.animateNextTransition();
  };

  const animate = () => {
    console.log('first time');
    ref.current.animateNextTransition();
  };
  const transition = (
    <Transition.Sequence>
      <Transition.Out type="fade" durationMs={300} interpolation="easeIn" />
      <Transition.Together>
        <Transition.In
          type="slide-right"
          durationMs={1000}
          interpolation="easeOut"
          propagation="bottom"
        />
      </Transition.Together>
    </Transition.Sequence>
  );

  return (
    <>
      <StatusBar barStyle="dark-content" />
      <SafeAreaView>
        <ScrollView
          contentInsetAdjustmentBehavior="automatic"
          style={styles.scrollView}>
          <View style={styles.body} />
          <Button
            title="refresh"
            color="#FF5252"
            onPress={() => handleRefreshed()}
          />
          <Transitioning.View ref={ref} transition={transition}>
            <View key={refreshed} style={styles.content}>
              <Text>Andi2</Text>
              <Text>Beni</Text>
              <Text>Candra</Text>
              <Text>Devi</Text>
              <Text>Yansen</Text>
            </View>
          </Transitioning.View>
        </ScrollView>
      </SafeAreaView>
    </>
  );
};

const styles = StyleSheet.create({
  scrollView: {
    backgroundColor: 'white',
  },
  engine: {
    position: 'absolute',
    right: 0,
  },
  body: {
    backgroundColor: 'white',
  },
  content: {
    flex: 1,
    flexGrow: 1,
    height: '100%',
    flexDirection: 'column',
  },
});

export default App;
```

**Comment by rafygonzalez** [comment]
The exact same thing happens to me, it is an error.

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.

We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
